src/cpp/session/modules/SessionShinyViewer.cpp

```diff
@@ -329,6 +329,15 @@ Error initShinyViewerPref(boost::shared_ptr<std::string> pShinyViewerType)
    {
       setShinyViewerType(*pShinyViewerType);
    }
+   
+   // the functions used to invoke the Shiny viewer changed in v1.2 -> v1.3;
+   // however, the older versions of these functions may be persisted as part
+   // of the R options (for shiny.launch.browser). for that reason, we detect
+   // if this option is set and ensure an up-to-date copy of the function is
+   // used as appropriate
+   Error error = r::exec::RFunction(".rs.refreshShinyLaunchBrowserOption").call();
```

Comment:
Should this be behind an `else`? (otherwise it looks like a NULL initial value will cause us to set the option twice)

Reply:
You're correct -- I've made that change.